src/content/en/begin-with/mask-editor.md

```
@@ -4,27 +4,33 @@ lang: en
 
-It features not only mask creation but also simple painting functions.
+It has not only mask creation but also a simple paint function.
+
+---
 
 ## How to Launch
 
-- Select a `Load Image` node, etc. → Click `🌔` (Open Mask Editor) in the `Node Selection Toolbox`.
+- Select `Load Image` node etc. → Click `🌔` (Open Mask Editor) in `Node Selection Toolbox`
 
 ![](https://gyazo.com/41526255834943bb591e62583d85d324){gyazo=loop}
```

markdownlint-cli2 (0.18.1)

Images should have alternate text (alt text)

(MD045, no-alt-text)